design/src/App.scss

```diff
@@ -188,3 +188,103 @@ $hover-color: #f8cc80;
         }
     }
 }
+
+
+
+// NAVBAR
+
+.navbar {
+  height: 96px;
+  background: #25271C;
+  position: sticky;
+
+  .navbar-content {
+    height: auto;
+    padding: 30px;
+    margin: 0 auto;
+    padding-top: 20px;
+
+    .navbar-logo {
+        float:left;
+    }
+    
+    .navbar-menu {
+        float:right;
+        position: relative;
+        top: 20px;
+    }
+  }
+}
+
+.menu-links {
+    list-style: none;
+
+    li {
+        display: inline-block;
+        margin-left: 36px;
+        font-family: Archivo;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 20px;
```

Comment:
Sätt gärna fontstorleken i `rem` eller `em` som enhet, för tillgänglighetens skull. Du kan skapa en funktion i Sass för det, så kan man mata in pixlar och få ut rem. Typ:

```
@function rem($px) {
  @return ($px / 16px) * 1rem;
}
```

Och använd som så: `font-size: rem(40px);`